style/ruby/README.md

```diff
@@ -3,12 +3,11 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
 * Avoid multiple assignments per line (`one, two = 1, 2`).
 * Avoid organizational comments (`# Validations`).
 * Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
   instead to emphasize code branches.
-* Avoid explicit return statements.
+* Don't use explicit `return` at the end of methods.
```

Comment:
Thoughts on modifying this to be "Don't use explicit `return` except for in guard clauses"? The current wording allows something like this:

```
def foo
  if foo.is_a? Integer
    return true
  else
    false
  end
end
```
(the explicit return is inside the conditional statement, not at the literal end of the method)